mpas_analysis/ocean/nino34_index.py

```diff
@@ -266,7 +321,7 @@ def _autocorr(x, t=1):  # {{{
     return np.corrcoef(np.array([x[0:len(x)-t], x[t:len(x)]]))  # }}}
 
 
-def _running_mean(inputData):  # {{{
+def _running_mean(inputData, wgts):  # {{{
     """
     Calculates 5-month running mean for NINO index and the spectra
```

Comment:
I think we may just want to say in the docstring here that this is a weighted (windowed) running mean so that the connection between the window and `wgts` is made in the docstring.